content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst

```diff
@@ -0,0 +1,233 @@
+============
+Product type
+============
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+
+Define *product types* in Odoo to track products in varying levels of detail.
+
+Classify products as *storable* to track stock counts, allowing users to trigger :doc:`reordering
+rules <../product_replenishment/reordering_rules>` for generating purchase orders. *Consumable*
+products are assumed to always be in stock, and *service* products are performed and served by the
+business.
+
+.. seealso::
+   `Odoo Tutorials: Product Type <https://www.youtube.com/watch?v=l6j0ZkP5mLM>`_
+
+Set product type
+================
+
+To set a product type, go to :menuselection:`Inventory app --> Products --> Product` and selecting
+the desired product.
+
+On the product form, in the :guilabel:`Product Type` field, select:
+
+- :guilabel:`Storable` for products tracked with stock counts. Only storable products can trigger
+  reordering rules for generating purchase orders
+
+.. tip::
+   Choose :guilabel:`Storable` if it is necessary to track a product's stock at various locations,
+   inventory valuation, or lots and serial numbers.
```

Comment:
```suggestion
   inventory valuations, or if the product has lots and serial numbers.
```
does that still ring true, with my rewording here?

Reply:
It does! Thank you for fixing up the language here (: